src/App.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
-import './App.css';
 
-const App = () => (
-  <h1>Count: 0</h1>
-);
+export class App extends React.Component {
+  state = {
+    count: 0,
```

Comment:
Переменные лучше называть существительными
```suggestion
    counter: 0,
```